examples/unix/c99/z_ping.c

```diff
@@ -0,0 +1,128 @@
+#include <pthread.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+
+#include "zenoh-pico.h"
+#include "zenoh-pico/api/primitives.h"
+#include "zenoh-pico/system/platform.h"
+
+_z_condvar_t cond;
```

Comment:
We shall consider making all these types public, if we plan to make use of them.
I might target it on a different PR to not pollute this one.

Reply:
Agreed, and zenoh-c should probably export the same condvar/mutex/malloc equivalents too :)

Reply:
well...then let's hold and do them at the same time for the after release IMO.